src/vos/vos_aggregate.c

```diff
@@ -1749,6 +1749,10 @@ vos_agg_ev(daos_handle_t ih, vos_iter_entry_t *entry,
 		return rc;
 	}
 
+	/* Aggregation Yield for testing purpose */
+	while (DAOS_FAIL_CHECK(DAOS_CONT_AGG_YIED))
+		ABT_thread_yield();
```

Comment:
It's better mark VOS_ITER_CB_YIELD by "*act |= VOS_ITER_CB_YIELD". (though your test doesn't update VOS tree while yielding)

Reply:
hmm ok, let me understand.

Reply:
I mean adding a "*act |= VOS_ITER_CB_YIELD" after this ABT_thread_yield().